### PR TITLE
add readme for emr vpc setup

### PR DESCRIPTION
**Pull request by RohitA5L**

### CODE REVIEW

**Comment by RohitA5L** [comment]
Would want to wait for @saguziel 's comments but if this is urgent @amargvela then you can merge it but maybe dont close the PR so alex can comment on it.

File: emr_sample/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+This repository contains terraform code that you can run as part of the Tecton cluster setup. This repository does two main things:
+
+* Sets up the VPC, Security Groups and Subnets that will be used by Tecton admins to deploy the cluster.
+* Sets up cross account roles that Tecton can use to setup, monitor and debug your cluster.
+
+You can run the following terraform commands in this folder which will execute the `infrastructure.tf` file.
+Run Terraform plan which will output the objects that will be created
+
+```
+aws-vault exec terraform -- terraform plan
```
Comment (saguziel) [inline emr_sample/README.md:12]: aws-vault exec terraform -- is specific to our setup. also they'd need to run `init`

File: emr_sample/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    This is AWS region where this cluster will be setup
+
+*  `account_id` :
+    This is the AWS account ID for your (customer) AWS Account
+
+* `is_vpc_deployment` :
+    Whether this is a VPC deployment. Default is set to true.
+
+* `elasticache_enabled` :
+    Whether you want to optionall also create a Redis cluster that Tecton will use for an online store. This currently defaults to False.
```
Comment (saguziel) [inline emr_sample/README.md:42]: typo here in optional

File: emr_sample/README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    **Note that Redis is currently in Beta and you should talk to Tecton before turning it on**
+
+* `tecton_dataplane_account_role_arn` :
+    Role used to run this terraform with. Usually the admin role in the account.
+
+* `ip_whitelist` :
+    IP ranges that should be able to access Tecton endpoint. Currently this defaults to `0.0.0.0/0` so everyone can access the Tecton endpoint
+
+* `tecton_assuming_account_id` :
+    This is the Tecton AWS account ID. Please get this from your Tecon rep.
```
Comment (saguziel) [inline emr_sample/README.md:52]: typo in tecton

**Pull request merged**